packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -30,26 +29,25 @@ import { getTelemetryReporter } from "../utilities/telemetry";
 import { DevicePlatform } from "../common/DeviceManager";
 import { isEasCliInstalled } from "../builders/easCommand";
 import { getMinimumSupportedNodeVersion } from "../utilities/getMinimumSupportedNodeVersion";
+import { LaunchConfiguration } from "../common/LaunchConfig";
 
 export class DependencyManager implements Disposable, DependencyManagerInterface {
-  private _appRootFolder: string;
   // React Native prepares build scripts based on node_modules, we need to reinstall pods if they change
   private eventEmitter = new EventEmitter();
   private packageManagerInternal: PackageManagerInfo | undefined;
 
-  constructor(appRootFolder: string) {
-    this._appRootFolder = appRootFolder;
-  }
+  constructor(private launchConfiguration: LaunchConfiguration) {}
 
   get appRootFolder() {
-    return this._appRootFolder;
+    return this.launchConfiguration.absoluteAppRoot;
```

Comment:
we can likely remove this entirely. Seems to be only used in this class and we can completely switch all the methods that take it to use LaunchConfiguration instead which should be cleaner (i.e. `isExpoGoProject` or `npmPackageVersionCheck`)